src/ui/public/vis/editors/components/editor_options_group.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './editor_options_group.less';
+
+import {
+  EuiAccordion,
+  EuiPanel,
+  EuiSpacer,
+  EuiTitle,
+  htmlIdGenerator,
+} from '@elastic/eui';
+
+
+/**
+ * A component to group different options in an editor together and give them
+ * a title. Should be used for all visualize editors when grouping options,
+ * to produce an aligned look and feel.
+ */
+function EditorOptionsGroup(props) {
+  return (
+    <EuiPanel
+      grow={false}
+      className="editorOptionsGroup__panel"
+    >
+      <EuiAccordion
+        id={htmlIdGenerator('eog')()}
+        initialIsOpen={true}
+        extraAction={props.actions}
+        buttonContent={
+          <EuiTitle size="xs">
+            <h2>{props.title}</h2>
```

Comment:
May want the consumer to be able to customize the heading level to maintain hierarchy for accessibility.

Reply:
I think currently we don't need that, since I wouldn't consider this grouping to be used nested within each other, but only as the high level grouping. Had some similar discussion with Peter about that, that if we have further nested elements (like aggregations or axis configuration) that might themselves be panels again, we should not use that component, but rather build an own component for that, and keep that component as the pure high level grouping. So I think `h2` should be fine for now. We could still change this later on, if we figure out we really need to change the level of the heading.